.github/workflows/gradle.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       uses: actions/setup-java@v2
       with:
         java-version: '11'
-        distribution: 'adopt'
+        distribution: 'zulu'
```

Comment:
To test two builds in parellel, this could be :
```yaml
         distribution: ${{ matrix.distribution }}
```
which could be set on the build:
```yaml
  build:
    runs-on: ubuntu-latest
    strategy:
      matrix:
        distribution: ['zulu', 'temurin' ]
```

Reply:
Cool.. I will do it now. sorry for the slow reply.

Reply:
Completed. It is nice that they complete near the same time. 👍🏼  
I also created a matrix for java-version with one version (11). As a convenience when jdk 17 becomes GA, it's easy to just add.